diff --git a/golpe/rules.mk b/golpe/rules.mk
index d8c83c2..c96c8d9 100644
--- a/golpe/rules.mk
+++ b/golpe/rules.mk
@@ -46,7 +46,7 @@ build/golpe.h: golpe/golpe.h.tt golpe/gen-golpe.h.pl $(wildcard global.h) $(wild
 %.d : ;
 
 build/defaultDb.h: $(wildcard golpe.yaml)
-	golpe/external/rasgueadb/rasgueadb-generate golpe.yaml build
+	perl golpe/external/rasgueadb/rasgueadb-generate golpe.yaml build
 
 clean:
 	rm -rf $(BIN) src/*.o src/*.d build/
